qcodes/instrument_drivers/ZI/ZIUHFLI.py

```diff
@@ -711,6 +710,22 @@ def __init__(self, name, device_ID, **kwargs):
                                vals=vals.Enum(*list(dmtrigs.keys()))
                                )
 
+            self.add_parameter('demod{}_sample'.format(demod),
+                               label='Demod sample',
+                               get_cmd=partial(self._getter, 'demods',
+                                               demod - 1, 2, 'sample'),
+                               snapshot_value=False
+                               )
+
+            for demod_param in ['x', 'y', 'R', 'phi']:
+
+                self.add_parameter('demod{}_{}'.format(demod, demod_param),
```

Comment:
Ideally, this should have a unit too. In the notebook, units are shown for these parameters (with a lowercase "v" instead of "V'), but where are they assigned?

Reply:
I must have forgotten to add the commit that does the units